mobile/src/libs/hooks/use-press-state/use-press-state.hook.ts

```diff
@@ -0,0 +1,46 @@
+import { AppColor } from '#libs/enums/enums';
```

Comment:
remove this, use android ripple instead

Reply:
I've tried using this before, but the problem is that it specifically changes the background, not the color. However, I just realized that I can use the activeOpacity property, and it looks something like this:

https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/108797614/34b707f7-e0bf-4c92-8f88-201b01603323


 What do you think about it?

Reply:
I like it